README.md

```diff
@@ -1,25 +1,24 @@
 # Receipt Data Analysis
 
-This project is about reading a receipt and using the data analysis to find patterns 
```

Comment:
data analysis is not a the thing

Reply:
agreed, that is why github is telling you that I removed it from this line